### PR TITLE
fix broken link in README

### PR DESCRIPTION
**Pull request by SvenDowideit**
... it in the Docker docs

Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

**Comment by vieux** [comment]
LGTM

File: userguide.md [hunk 1]
```diff
@@ -101,5 +101,4 @@ See the [Discovery service](../discovery) document for more information.
 
 ## Advanced Scheduling
 
-See [filters](../scheduler/filter) and [strategies](../scheduler/strategy) to learn
```
Comment (vieux) [inline userguide.md]: Should we do the same change in the main README.md ?

**Comment by vieux** [comment]
@SvenDowideit please merge #361and update this one after for `userguide.md`

**Comment by SvenDowideit** [comment]
awesome, will do!

**Comment by SvenDowideit** [comment]
much better :)

**Pull request merged**